cypher/cypher-docs/src/docs/dev/ql/administration/security/dbms/admin-role-dbms.asciidoc

```diff
@@ -4,10 +4,10 @@ These include:
 * Create or drop databases
 * Change configuration parameters
 * Manage transactions
-* Manage users and roles (role management by itself is assignable using <<administration-security-administration-dbms-privileges-role-management, Cypher commands>>)
+* Manage users and roles
```

Comment:
The description of this list is `the `admin` role has a number of built-in privileges that cannot be assigned using Cypher commands. These include:` so maybe the description  should be changed?

Reply:
Good point